cooltools/api/expected.py

```diff
@@ -1035,6 +1035,10 @@ def per_region_smooth_cvd(
     )
 
     return cvd
+
+def _transformer(p, weight1, weight2):
```

Comment:
I wonder if this might be better named `_balance_transform`

Reply:
Yea, that might be more informative. Updated